account-compression/programs/account-compression/src/canopy.rs

```diff
@@ -0,0 +1,127 @@
+//! Canopy is way to cache the upper `N` levels of a SPL ConcurrentMerkleTree.
```

Comment:
This is great, thanks for providing the info

Reply:
👍